### ISSUE TITLE
Notifier configurable class names

### ISSUE THREAD

**Issue by chay22** [MohammadYounes/AlertifyJS#217]
I was trying to use `alertify.success()` with no luck. And found out it's being blocked by [Adblocker for Chrome - NoAds](https://chrome.google.com/webstore/detail/adblocker-for-chrome-noad/alplpnakfeabeiebipdmaenpmbgknjce) extension (I think it's not currently available on Chrome store anymore?). A simple Google search also shows me an adblock filter list that includes Alertify selector to hide, `.ajs-message`, [here](https://github.com/uBlock-LLC/uBlock/blob/master/assets/thirdparties/secure.fanboy.co.nz/fanboy-cookiemonster.txt#L6690).

**Comment by chay22** [comment]
I ended up override notification functions as workaround. Something like,
````
import alertify from 'alertifyjs'

['success', 'error', 'message', 'warning', 'notify'].forEach((v) => {
  const original = alertify[v]

  alertify[v] = function () {
    const instance = original.apply(this, arguments)

    instance.element.classList.remove('ajs-message')
    instance.element.classList.add('another-class-name')

    return instance
  }
})
````

I think it's better to just override than try to implement on alertify. Closing it.

**Comment by MohammadYounes** [comment]
This is not recommended, as it might break in future updates ... I believe it would be good to add it as enhancement.

Thanks!

**Comment by MohammadYounes** [comment]
Closed in 792c02dd2877b7254aba09f18c15e536eb00ed4d. See https://alertifyjs.com/guide.html#defaults
